.circleci/config.yml

```diff
@@ -186,6 +186,7 @@ jobs:
       - image: circleci/node:10.16-browsers
     steps:
       - checkout
+      - run: sudo apt-get install shellcheck
```

Comment:
I don't especially like installing things during CI workflows 🤔 

Maybe we could use the `shellcheck` orb instead, or the docker image? We wouldn't be using the `yarn:shellcheck` script in that case though... not ideal, as it could get out of sync with CI.

If we wanted to use `yarn:shellcheck` directly, we could either use the shellcheck npm package, or we could create our own Docker based upon `circleci/node:10.16-browsers` (or forked from their Dockerfile) that includes shellcheck.

Hmm. Maybe this is fine for now, and we can remove this step later when we create our own Dockerfile. I had wanted to do that in the near future anyway, to make it easier to test on a range of browser versions. Unless you wanted to try one of these alternatives.